core/auth/auth.go

```diff
@@ -2383,8 +2391,8 @@ func (l *LocalDataLoaderImpl) LoadServices() ([]authservice.ServiceReg, error) {
 	return authRegs, nil
 }
 
-//NewLocalDataLoader creates and configures a new LocalDataLoaderImpl instance
-func NewLocalDataLoader(storage Storage) *LocalDataLoaderImpl {
+//NewLocalServiceRegLoader creates and configures a new LocalServiceRegLoaderImpl instance
+func NewLocalServiceRegLoader(storage Storage) *LocalDataLoaderImpl {
```

Comment:
We can delete the `GetAccessToken` and `GetDeletedAccounts` functions here, since they are not part of the `ServiceRegLoader` interface. I think we should update the name of `LocalDataLoaderImpl` for consistency as well.